Bulldozer.BinaryFile/Maps/PersonDocument.cs

```diff
@@ -197,6 +196,12 @@ public int ProcessFiles( List<ZipArchiveEntry> importFiles, RockContext rockCont
                     continue;
                 }
 
+                if ( parsedFileName.Length > 6 )
+                {
+                    errors += $"{DateTime.Now}, Binary File Import, Filename '{file.Name}' is not in expected format. It has more segments than expected. Result of importing this file may be unexpected.\r\n";
+                    continue;
```

Comment:
Should we break or just warn them and still try to import?

Reply:
```suggestion
```